source/braille.py

```diff
@@ -1205,7 +1205,12 @@ def _addTextWithFields(self, info, formatConfig, isSelection=False):
 				elif cmd == "controlEnd":
 					# Exiting a controlField should break a run of clickables
 					inClickable=False
-					field = ctrlFields.pop()
+					# Index error may occur with Teraterm in non-graphical Linux.

```

Comment:
This change doesn't fix a bug but rather obfuscates it. There might be something wrong in DisplayModelTextInfo that causes it (i.e. i assume Teraterm uses the display model).

Reply:
I am about to revert this. New issue some day if needed.